NetKAN/SVT-MacLinux.netkan

```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?i).Mac.Linux.zip",
```

Comment:
`"#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*Mac.Linux.zip"` would suffice and be less confusing.